DSA-Campaign-Uplift-Estimation/src/main/webapp/Create/create-script.js

```diff
@@ -292,9 +292,15 @@ function sendFormData() {
   // default values for variables (not applicable to creation phase) sent to servlet
   keyvalPairs.push(encodeURIComponent('DSACampaignId') + '=' + encodeURIComponent('0'));
   keyvalPairs.push(encodeURIComponent('campaignStatus') + '=' + encodeURIComponent('pending'));
-  keyvalPairs.push(encodeURIComponent('clicks') + '=' + encodeURIComponent('0'));
-  keyvalPairs.push(encodeURIComponent('cost') + '=' + encodeURIComponent('0'));
-  keyvalPairs.push(encodeURIComponent('impressions') + '=' + encodeURIComponent('0'));
+
+  /* temporary random generation for data */
+  let clicks = Math.ceil(Math.random() * 500000);
+  let cost = Math.random() * 23000;
+  let impressions = Math.ceil(Math.random() * 40000);
```

Comment:
Can you do this instead -
let impressions = Math.round(1000 + Math.random()*1000);
let clicks = Math.round(250 + Math.random()*250);
let cost = Math.round(100 + Math.random()*100);
I think these are more accurate daily estimates (impressions should be > clicks). Also, for display purposes the numbers should be closer to each other.

Reply:
Sounds good.